SofaKernel/modules/SofaBaseCollision/SofaBaseCollision_test/BroadPhase_test.h

```diff
@@ -84,7 +83,17 @@ struct MyBox{
 };
 
 template <class BroadPhase>
-struct BroadPhaseTest: public ::testing::Test{
+struct BroadPhaseTest: public ::testing::Test
+{
+    void SetUp() override
+    {
+        // force load SofaMiscCollision
+        sofa::component::initSofaMiscCollision();
+    }
+    void TearDown() override
+    {
+    }
```

Comment:
empty you are sure? no need to clean graph?

Reply:
if (root)
            Simulation::theSimulation->unload(root);

Reply:
there is no created simulation actually in those tests.